### PR TITLE
Update slim images to include swift-backtrace.

### PR DESCRIPTION
**Pull request by al45tair**
The slim images should include swift-backtrace as well as the runtime libraries.

rdar://112966082

### CODE REVIEW

**Comment by al45tair** [comment]
@swift-ci test

**Comment by al45tair** [comment]
@swift-ci test

**Review by tomerd — APPROVED** [review]

**Review by shahmishal — COMMENTED** [review]
We should also apply these changes to swift-ci/master/ Dockerfiles, it will help us get backtrace for CI failures.

File: nightly-5.9/centos/7/slim/Dockerfile [hunk 1]
```diff
@@ -34,7 +34,9 @@ RUN set -e; \
     && curl -fSsL https://swift.org/keys/all-keys.asc | gpg --import -  \
     && gpg --batch --verify latest_toolchain.tar.gz.sig latest_toolchain.tar.gz \
     # - Unpack the toolchain, set libs permissions, and clean up.
-    && tar -xzf latest_toolchain.tar.gz --directory / --strip-components=1 ${DOWNLOAD_DIR}-${OS_VER}/usr/lib/swift/linux \
+    && tar -xzf latest_toolchain.tar.gz --directory / --strip-components=1 \
+       ${DOWNLOAD_DIR}-${OS_VER}/usr/lib/swift/linux \
+       ${DOWNLOAD_DIR}-${OS_VER}/usr/libexec/swift/linux \
     && chmod -R o+r /usr/lib/swift \
```
Comment (benlangmuir) [inline nightly-5.9/centos/7/slim/Dockerfile:40]: Missing`chmod` for `/usr/libexec/swift`? The other Dockerfiles all added that.

**Comment by al45tair** [comment]
> We should also apply these changes to swift-ci/master/ Dockerfiles, it will help us get backtrace for CI failures.

FWIW, looking at the swift-ci docker files, _assuming_ the `chmod` is not really necessary, I don't think there's anything to do there.

**Pull request merged**